src/routes/login/LoginView.vue

```diff
@@ -1,393 +1,393 @@
+            </p>
+
+            <v-form
+              id="login"
+              v-model="isFormValid"
+              class="mx-auto loginForm"
+              @submit.prevent="submitLogin"
+            >
+              <label for="username" class="text-body-1 font-weight-bold">{{ t('global.username') }}</label>
+              <v-text-field
+                id="username"
+                ref="usernameInput"
+                v-model="username"
+                class="mt-4"
+                variant="solo"
+                :rules="usernameRules"
+                :density="$vuetify.display.mdAndDown ? 'compact' : 'default'"
+                autocomplete="username"
+                data-cy="username"
+              />
+
+              <label for="password" class="text-body-1 font-weight-bold">{{ t('global.password') }}</label>
+              <v-text-field
+                id="password"
+                v-model="pw"
+                class="my-4"
+                variant="solo"
+                :rules="passwordRules"
+                :density="$vuetify.display.mdAndDown ? 'compact' : 'default'"
+                :type="showPass ? 'text' : 'password'"
+                :autocomplete="isLoggingIn ? 'current-password' : 'new-password'"
+                :append-inner-icon="showPass ? 'mdi-eye' : 'mdi-eye-off'"
+                data-cy="password"
+                @click:append-inner="showPass = !showPass"
+              />
+
+              <div
+                id="login-button-container"
+                class="d-flex flex-column flex-md-row justify-space-between align-center flex-wrap"
+              >
+                <v-btn
+                  class="px-16"
+                  :loading="loading"
+                  :disabled="!isFormValid"
+                  :color="isFormValid ? 'newPrimary' : 'disabled'"
+                  type="submit"
+                  size="x-large"
+                  text-color="white"
+                  data-cy="submit"
+                >
+                  {{ buttonText }}
+                </v-btn>
+                <v-btn
+                  class="px-0 text-h6"
+                  color="black"
+                  variant="text"
+                  data-cy="switch-mode"
+                  @click="switchMode"
+                >
+                  {{ switchLabelText }}
+                </v-btn>
+              </div>
+
+              <v-btn
+                class="w-100 my-10 text-subtitle-2 text-sm-subtitle-1 text-md-h6 h-auto py-2"
+                size="large"
+                color="newSecondary"
+                href="https://discord.com/invite/9vrAZ8xGyh"
+                target="_blank"
+              >
+                <img class="discord" src="/img/loginView/logo-discord.svg">
+                <span class="discordButton">{{ t('login.joinDiscord') }}</span>
+              </v-btn>
+            </v-form>
+
+            <BaseSnackbar
+              v-model="showSnackBar"
+              :message="t(snackBarMessage)"

```

Comment:
atm the variable `snackBarMessage` isn't actually the snackbar message, it's the key to that message. Let's move the `t()` call into the function that _sets_ `snackBarMessage` so that this can be simpler here
```suggestion
              :message="snackBarMessage"

```